libs/SmartSync/SmartSyncTests/ParentChildrenSyncTests.m

```diff
@@ -1189,6 +1189,93 @@ - (void)testSyncUpLocallyDeletedParentRemotelyDeletedParentNoChildren {
     [self trySyncUpsWithVariousChangesWithNumberAccounts:2 numberContactsPerAccount:0 localChangeForAccount:DELETE remoteChangeForAccount:DELETE localChangeForContact:NONE remoteChangeForContact:NONE];
 }
 
+/**
+ * Create accounts and contacts on server, sync down
+ * Update some of the accounts and contacts - using bad names (too long) for some
+ * Sync up
+ * Check smartstore and server afterwards
+ */
+- (void) testSyncUpWithErrors
```

Comment:
New test for regular sync up.
Notice we are syncing up multiple good and bad records. If we were still stopping at the first error, the test would fail.